src/current/cockroachcloud/resource-usage-basic.md

```diff
@@ -0,0 +1,191 @@
+
+### Expensive queries
+
+Expensive queries, [especially `FULL SCAN` operations]({% link {{site.current_cloud_version}}/performance-best-practices-overview.md %}#table-scan-best-practices), are the most common cause of unexpected RU consumption increases and a good place to begin investigating your consumption. To diagnose expensive queries:
+
+1. Navigate to the [**Statements** tab]({% link cockroachcloud/statements-page.md %}) of your cluster's **SQL Activity** page in the {{ site.data.products.cloud }} Console.
+1. Click on the title of the **Rows Processed** column to sort your queries by the number of rows processed. 
+ 
+    For most queries, total rows processed should be no more than a few hundred. Read queries are often more expensive than write queries.
+  
+1. Next, sort the queries by the **Bytes Read** column. Most queries should read no more than a few kilobytes per row.
+
+If any queries are more expensive than expected, you can use the [`EXPLAIN ANALYZE` SQL command]({% link {{site.current_cloud_version}}/explain-analyze.md %}#global-properties) for an estimate of the RUs consumed. Efficient queries generally consume fewer RUs, so the guidelines for [Optimizing Query Performance](https://www.cockroachlabs.com/docs/stable/performance-best-practices-overview) can be applied here. You can also refer to [Cockroach Labs Blog - How to troubleshoot and optimize query performance in CockroachDB](https://www.cockroachlabs.com/blog/query-performance-optimization/) for further information.
+
+### Excessive queries
+
+Each query has an associated RU cost, so the total number of queries is an important factor in your consumption.
+
+To diagnose excessive queries, navigate to your cluster's **Metrics** page, [**Monitor SQL Activity** tab]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) in the {{ site.data.products.cloud }} Console. The **SQL Statements** chart displays the number of queries over time. Look for any spikes or increases in QPS (queries per second) that may correspond to increases in your resource consumption.
+
+Reducing the rate of queries is application-specific and must be achieved at the application level.
+
+### Excessive number of connections
+
+CockroachDB {{ site.data.products.basic }} clusters consume minimal resources per connection, so the number of connections is not the most likely cause of increased RU consumption. However, managing your connections is increasingly important for both performance optimization and RU consumption as your application scales up.
```

Comment:
```suggestion
CockroachDB {{ site.data.products.basic }} clusters consume minimal resources per connection, so increased RU consumption is not likely to be caused by a high number of connections. However, it will be important to manage your connections for both performance optimization and RU consumption as your application scales up.
```